--- a/feeds/packages/utils/tmux/Makefile
+++ b/feeds/packages/utils/tmux/Makefile
@@ -12,7 +12,8 @@ PKG_VERSION:=1.9a
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/tmux
+#PKG_SOURCE_URL:=@SF/tmux
+PKG_SOURCE_URL:=https://github.com/tmux/tmux/releases/download/$(PKG_VERSION)
 PKG_MD5SUM:=b07601711f96f1d260b390513b509a2d
 PKG_MAINTAINER:=Maxim Storchak <m.storchak@gmail.com>
 
